modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -25,14 +28,46 @@ The `high-limit` format allows more nodes and relationships to be stored than `s
 
 You can either set the store format when creating a new database or change the store format of an existing database.
 
+The default store format of new databases is `aligned` and is controlled by the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration setting in the _neo4j.conf_ file.
```

Comment:
Why move this text up from the **Creating new databases** where it was? That setting controls format for **_new_** databases only

Reply:
You are right, but my idea is that by the config you set a default format for all new databases. And the heading below is about a creating a single database on a specific format.